lib/collection/src/operations/payload_ops.rs

```diff
@@ -4,22 +4,28 @@ use serde;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+#[derive(Debug, Deserialize, Serialize, JsonSchema)]
+pub struct SetPayload {
+    pub payload: HashMap<PayloadKeyType, PayloadInterface>,
+    /// Assigns payload to each point in this list
+    pub points: Vec<PointIdType>,
+}
+
+#[derive(Debug, Deserialize, Serialize, JsonSchema)]
+pub struct DeletePayload {
+    pub keys: Vec<PayloadKeyType>,
+    /// Deletes values from each point in this list
+    pub points: Vec<PointIdType>,
+}
+
 /// Define operations description for point payloads manipulation
 #[derive(Debug, Deserialize, Serialize, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum PayloadOps {
     /// Set payload value, overrides if it is already exists
-    SetPayload {
-        payload: HashMap<PayloadKeyType, PayloadInterface>,
-        /// Assigns payload to each point in this list
-        points: Vec<PointIdType>,
-    },
+    SetPayload(SetPayload),
     /// Deletes specified payload values if they are assigned
-    DeletePayload {
-        keys: Vec<PayloadKeyType>,
-        /// Deletes values from each point in this list
-        points: Vec<PointIdType>,
-    },
+    DeletePayload(DeletePayload),
     /// Drops all Payload values associated with given points.
     ClearPayload { points: Vec<PointIdType> },
```

Comment:
Could we update this operation in similar way as others, so the payload would look like
```json
{
  "points": [1,2,3]
}
```
It is also possible, that would want to use payload query as a selector instead of id list.

Reply:
WDYT about https://github.com/qdrant/qdrant/pull/221/commits/ba22662a6f2f70a7aeadae386c5e0eb71b9f5b2f

Reply:
:+1: